cypress/page/products-list.page.ts

```diff
@@ -3,8 +3,8 @@ class ProductsListPage {
   private checkout: string;
 
   constructor (){
-    this.addToCartButton = "#center_column a.button.ajax_add_to_cart_button.btn.btn-default";
-    this.checkout = "[style*='display: block;'] .button-container > a";
+    this.addToCartButton = ".ajax_add_to_cart_button > span";
```

Comment:
you can use an attribute here to indicate we are clicking on the `Add to cart` button